deployment/constructor_params/mainnet/bqeth.yml

```diff
@@ -0,0 +1,41 @@
+deployment:
+  name: bqeth-mainnet
+  chain_id: 137
+
+artifacts:
+  dir: ./deployment/artifacts/
+  filename: bqeth-mainnet.json
+
+constants:
+  # See deployment/artifacts/mainnet.json
+  COORDINATOR_PROXY: "0xE74259e3dafe30bAA8700238e324b47aC98FE755"
+  
+  BQETH_ADOPTER: "TBD"
+
+  # DAI Token on Polygon PoS - References:
+  # - https://polygonscan.com/token/0x8f3cf7ad23cd3cadbd9735aff958023239c6a063
+  # - https://github.com/search?q=0x8f3Cf7ad23Cd3CaDbD9735AFf958023239c6A063&type=code
+  DAI_ON_POLYGON: "0x8f3Cf7ad23Cd3CaDbD9735AFf958023239c6A063"
+
+  # See https://github.com/nucypher/tdec/issues/169
+  BASE_FEE_RATE: 4050925925925  # $0.35 per day, in DAI units per second (in Python: 35*10**16 // 86400)
+  ENCRYPTOR_FEE_RATE: 63419583967  # $2 per year, in DAI units per second (in Python: 2 * 10**18 // 86400 // 365)
+  MAX_NODES: 30
```

Comment:
These parameters are correct. I do want to note that we will need to manually increase the `BASE_FEE_RATE` in one year. This was agreed to by the adopter. 

I.e. in the billing period after next, it should be $0.3675 per node per day ($2,012.065 for 30 nodes / 6 months).

Reply:
Yep, but for the moment we'll address this as a contract upgrade. Opened https://github.com/nucypher/nucypher-contracts/issues/281 to track this anyway.

Reply:
Update: There's already a PR for this (#283), and this branch is already on top of that one.